com.sap.cloud.lm.sl.cf.core/src/test/java/com/sap/cloud/lm/sl/cf/core/cf/clients/ServiceWithAlternativesCreatorTest.java

```diff
@@ -48,62 +38,44 @@
                 Arguments.of("service-03.json", "Service label must not be null", IllegalArgumentException.class),
                 Arguments.of("service-04.json", "Service name must not be null", IllegalArgumentException.class),
                 Arguments.of("service-05.json", "Service plan must not be null", IllegalArgumentException.class),
-                Arguments.of("service-06.json", "Could not create service instance \"foo\". Service plan \"v3.4-extra-large\" from service offering \"mongodb\" was not found.", CloudOperationException.class),
-                Arguments.of("service-07.json", "Could not create service instance \"foo\". Service plan \"v2.0-sp3\" from service offering \"hana\" was not found.", CloudOperationException.class),
+                Arguments.of("service-06.json", "401 Unauthorized", CloudOperationException.class),
```

Comment:
What about the detailed message? Is it gone and how the 401 Unauthorized is replacing it?

Reply:
It is a totally different test, in fact test case 16 and 17 are in place 06 and 07. Test cases 06 and 07 are removed because they are not applicable any more. The logic, which they tested is in CloudControllerClient implementation.

Reply:
Okay